### PR TITLE
[Zhang Dongjun] More implementations on the quiz mode command

### PR DESCRIPTION
**Pull request by Auxinnn**
Add an operation to store questions in a quiz question list, can be merged after reviewed.

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -15,19 +18,23 @@
...
      */
+
     {
         notes = new UniqueNoteList();
+        quiz = new QuizQuestionList();
     }
 
-    public AddressBook() {}
+    public AddressBook() {
+    }
```
Comment (Parcly-Taxel) [inline src/main/java/seedu/address/model/AddressBook.java:37]: Null constructors should be left out, as they are in the existing classes.

**Review by Parcly-Taxel — COMMENTED** [review]
Otherwise it is fine.

File: src/main/java/seedu/address/model/AddressBook.java (hunk 1 above)
Comment (Auxinnn) [reply src/main/java/seedu/address/model/AddressBook.java:37]: Do you want me to remove it or make it as the original form?

Comment (Parcly-Taxel) [reply src/main/java/seedu/address/model/AddressBook.java:37]: Eh, never mind. This is too little a detail to squirm over. Merge your pull request.

**Pull request merged**